mnefun/_mnefun.py

```diff
@@ -1642,58 +1724,19 @@ def apply_preprocessing_combined(p, subjects, run_indices):
             raw = _raw_LRFCP(r, None, p.hp_cut, p.lp_cut, p.n_jobs_fir,
                              p.n_jobs_resample, projs, bad_file,
                              disp_files=False, method='fft', apply_proj=False,
-                             filter_length=p.filter_length, force_bads=False)
+                             filter_length=p.filter_length, force_bads=False,
+                             l_trans=p.hp_trans, h_trans=p.lp_trans)
             raw.save(o, overwrite=True)
         # look at raw_clean for ExG events
         if p.plot_raw:
             _viz_raw_ssp_events(p, subj, run_indices[si])
 
 
-def gen_layouts(p, subjects):
```

Comment:
Yeah, mne-python has something that does this. I also doubt it was used by anyone other than me.